dephell/commands/generate_license.py

```diff
@@ -26,6 +26,7 @@ def get_parser(cls) -> ArgumentParser:
         builders.build_config(parser)
         builders.build_output(parser)
         builders.build_other(parser)
+        parser.add_argument('owner', help='license owner')
```

Comment:
You've specified it in `builders.build_other`. So, you don't need to specify it here. Remove, please.

Reply:
Removed this.